static/redfish/v1/$metadata/index.xml

```diff
@@ -3743,6 +3743,9 @@
         <edmx:Include Namespace="OemLogEntry"/>
         <edmx:Include Namespace="OemLogEntry.v1_0_0"/>
     </edmx:Reference>
+    <edmx:Reference Uri="/redfish/v1/schema/OemLogEntryAttachment_v1.xml">
+        <edmx:Include Namespace="OemLogEntryAttachment"/>
+    </edmx:Reference>
```

Comment:
Don't need 
"OemLogEntryAttachment.v1_0_0" like the others have?

Reply:
Some of the others used just '.v1', like OemComputerSystem_v1.xml
I guess I don't which is the rule here '.v1' vs. '.v1_0_0'

Reply:
Ack.